examples/tutorials/hello-gltf/app.ts

```diff
@@ -15,9 +15,15 @@ import {Matrix4} from '@math.gl/core';
 export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
   device: Device;
   scenes: GroupNode[] = [];
+  animator?: GLTFAnimator;
   center = [0, 0, 0];
   vantage = [0, 0, 0];
   time: number = 0;
+  options: Record<string, boolean> = {
+    'Camera-Animation': true,
+    'glTF-Animation': false,
+    PBR: false
+  };
```

Comment:
- Do we want such awkward keys? are we copying them directly from some glTF without checking? Otherwise call them `cameraAnimation`, `gltfAnimation`, `pbr`?
- If this is a "bag" of unknown options, maybe call it something more specific (gltfOptions)?
- if not, maybe type the the options so we don't erase the knowledge about valid keys:
```suggestion
  options: {
    'Camera-Animation': boolean;
    'glTF-Animation': boolean;
    PBR: boolean;
  } = {
    'Camera-Animation': true,
    'glTF-Animation': false,
    PBR: false
  }
```